### ISSUE TITLE
Unable to install a font bundle

### ISSUE THREAD

**Issue by jenneth519** [keremciu/sketch-iconfont#171]
I exported a font-bundle on the latest version (4.7.0) but then had to reinstall my plugins. If I try to install that font-bundle or any other one, nothing happens. I am able to install a font by SVG.

**Comment by ketchupbear** [comment]
The same thing is happening to me. I am running Sketch 54.0.

**Comment by ketchupbear** [comment]
@jenneth519 I was pointing to the folder wrong. I followed this video, selected the parent folder "font-bundles-master" and it worked.
[http://recordit.co/2PMTB04Jav](http://recordit.co/2PMTB04Jav)

**Comment by jenneth519** [comment]
I tried what was in the video and that wasn't even working for me. However, I updated to Sketch to 54.1 today and tried again. At first, it wouldn't install my custom bundle (nothing happened when I selected the "bundle" folder or the parent folder of "bundle"). But then I tried selecting the parent folder of the prepackaged "font-bundles-master" and I got the notification it worked. But now, when I check out my icon fonts in the "grid insert" menu - I see whats in my custom bundle PLUS the prepackaged. Some of which are duplicates - which I guess is better than none at all!
